cli/src/commands/workspace/import.rs

```diff
@@ -46,13 +53,99 @@ pub async fn main(args: Args) -> anyhow::Result<()> {
         )
         .await?;
 
+    let (notify, _event_conn) =
+        notify_sync_completion(&client.event_bus, wid, files_to_sync.clone());
+
+    copy_file_to_fd(src, &workspace, fd).await?;
+    log::debug!("Flushing and closing file");
+    workspace.fd_flush(fd).await?;
+    workspace.fd_close(fd).await?;
+
+    if !files_to_sync.lock().expect("Mutex poisoned").is_empty() {
```

Comment:
This might be the best place to add a comment. Explain for example _why_ there might many be files to sync, _why_ we need to wait, etc.

The comment below is only an example, adapt it if it is not right. 
```suggestion
    // There maybe other files to sync besides the one we want to import.
    // We wait until everything is synced.
    if !files_to_sync.lock().expect("Mutex poisoned").is_empty() {
```